controller/support_bundle_controller.go

```diff
@@ -687,6 +689,14 @@ func (c *SupportBundleController) newSupportBundleManager(supportBundle *longhor
 									Name:  "SUPPORT_BUNDLE_COLLECTOR",
 									Value: "longhorn",
 								},
+								{
+									Name:  "SUPPORT_BUNDLE_NODE_SELECTOR",
+									Value: c.getNodeSelectorString(nodeSelector),
+								},
+								{
+									Name:  "SUPPORT_BUNDLE_TAINT_TOLERATION",
+									Value: c.getTaintTolerationString(),
```

Comment:
Suggest using `tolerations` like `c.getTaintTolerationString(tolerations)` instead of getting the system setting again.